server/src/main/scala/org/labrad/browser/Controllers.scala

```diff
@@ -1,21 +0,0 @@
-package org.labrad.browser
-
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
-import play.api.libs.json._
-import play.api.mvc._
-import scala.concurrent.Future
-
-trait JsonRpc { self: Controller =>
-
-  def rpc[A: Reads, B: Writes](f: A => Future[B]) = Action.async(BodyParsers.parse.json) { request =>
-    val originOpt = request.headers.get("Origin")
-    val a = request.body.as[A]
-    f(a).map { b =>
-      val headers = Seq.newBuilder[(String, String)]
-      for (origin <- originOpt) {
-        headers += "Access-Control-Allow-Origin" -> origin
-      }
-      Ok(Json.toJson(b)).withHeaders(headers.result: _*)
-    }
-  }
-}
```

Comment:
So has this just been broken out into separate controllers for each sub-application? When I look in the folder now, there's no Controllers.scala, but theres the WebsocketController and a bunch of other ones.